src/main/java/study/racing/RacingGame.java

```diff
@@ -0,0 +1,36 @@
+package study.racing;
+
+import study.racing.strategy.MoveUnderOneDigitStrategy;
+
+import java.util.List;
+import java.util.Map;
+
+public class RacingGame {
+
+    public void racing() {
+
+        InputView inputView = new InputView();
+
+        int carCount = 0;
+        int tryCount = 0;
+
+        while(true) {
+            carCount = inputView.input(RacingMessage.CAR_COUNT.getCode());
+            tryCount = inputView.input(RacingMessage.TRY_COUNT.getCode());
+
+            boolean isInputMatch = (carCount != 0 && tryCount != 0);
+            if(!isInputMatch) {
+                System.out.println(RacingMessage.TRY_COUNT.getMsg());
+                return;
+            }
+            break;
+        };
+
+        PrepareForRacing prepareForRacing = new PrepareForRacing(new MoveUnderOneDigitStrategy());
+        List<Map<Integer, Integer>> racingList = prepareForRacing.personalCarTotalCount(tryCount, carCount);
+
+        for(int i= 0; i < racingList.size(); i++) {
+            ResultView.printView(racingList.get(i));
+        }
+    }
```

Comment:
해당 계층은 Controller의 역할과 Service의 역할이 같이 있는 것으로 보여요. 다음과 같은 모습으로 만들어보면 어떨까요? 


```suggestion
    public void racing() {
        InputView inputView = new InputView();
        OutputView outputView = new OutputView();
        RacingResult result = racingGameService.startRace(inputView.getRound(), inputView.getCars());
        
        outputView.showResult(result);
    }
```

이런식으로 간결하게 작성할 수 있을 것 같아요. 
그리고 service에서 startRace가 해야 할 역할도 간결하게 작성해보면 이런식이 되지 않을까요? 

```java
public RacingResult startRace(int rount, int carSize) {
    List<Car> cars = makeCars(carSize);
    List<RacingHistory> racingHistories = new ArrayList<>();
 
    for (int i =0; i< round; i++) {
        RacingHistory history = notifyMoveEvent(cars);
        racingHistory.add(history);
    }

    return RacingResult.from(racingHistory);
}

```